src/RegistryCoordinator.sol

```diff
@@ -732,6 +733,14 @@ contract RegistryCoordinator is
         stakeRegistry.initializeQuorum(quorumNumber, minimumStake, strategyParams);
         indexRegistry.initializeQuorum(quorumNumber);
         blsApkRegistry.initializeQuorum(quorumNumber);
+        // Check if the AVS has migrated to operator sets
+        AVSDirectory avsDirectory = AVSDirectory(serviceManager.avsDirectory());
+        if (avsDirectory.isOperatorSetAVS(address(serviceManager))){
```

Comment:
Assume we do this check because we don't want the migration function to fail?

Reply:
I think we could createOperatorSets before migration before, so I wanted to prevent that at the time